installer_scripts/install-yb-voyager

```diff
@@ -248,41 +248,54 @@ macos_main() {
 # COMMON
 #=============================================================================
 
-# Function to fetch the latest release data from GitHub API
-fetch_latest_release_data() {
-    # Fetch the latest release data from GitHub API
-    LATEST_RELEASE_DATA=$(curl -s https://api.github.com/repos/yugabyte/yb-voyager/releases/latest)
-    if [ -z "$LATEST_RELEASE_DATA" ]; then
-        echo "ERROR: Failed to fetch the latest release data from the GitHub API."
-        exit 1
-    fi
+# Function to fetch the release data from GitHub API
+fetch_release_data() {
+    # Fetch the latest release data from GitHub API if VERSION is latest
+	if [ "${VERSION}" == "latest" ]; then
+		RELEASE_DATA=$(curl -s https://api.github.com/repos/yugabyte/yb-voyager/releases/latest)
+		if [ -z "$RELEASE_DATA" ]; then
+			echo "ERROR: Failed to fetch the latest release data from the GitHub API."
+			exit 1
+		fi
+	fi 
 
-    # Extract the latest release name and tag name
-    LATEST_RELEASE_NAME=$(echo "$LATEST_RELEASE_DATA" | jq -r '.name')
-    LATEST_TAG_NAME=$(echo "$LATEST_RELEASE_DATA" | jq -r '.tag_name')
+    # Extract the latest release name and tag name from the fetched data if VERSION is latest
+	if [ "${VERSION}" == "latest" ]; then
+		RELEASE_NAME=$(echo "$RELEASE_DATA" | jq -r '.name')
+		TAG_NAME=$(echo "$RELEASE_DATA" | jq -r '.tag_name')
+	else 
+		# If the version is not latest, then the provided VERSION is used in the release name and tag name
+		RELEASE_NAME="v${VERSION}"
+		TAG_NAME="yb-voyager/v${VERSION}"
+	fi
 
-    # Fetch the commit hash of the latest tagged commit
-    LATEST_TAG_DATA=$(curl -s https://api.github.com/repos/yugabyte/yb-voyager/git/refs/tags/${LATEST_TAG_NAME})
-    if [ -z "$LATEST_TAG_DATA" ]; then
-        echo "ERROR: Failed to fetch the latest tagged commit data from the GitHub API."
+    # Fetch the commit hash of the tagged commit related to the release
+    TAG_DATA=$(curl -s https://api.github.com/repos/yugabyte/yb-voyager/git/refs/tags/${TAG_NAME})
+    if [ -z "$TAG_DATA" ]; then
+        echo "ERROR: Failed to fetch the tagged commit data from the GitHub API."
         exit 1
     fi
 
-    LATEST_TAGGED_COMMIT=$(echo "$LATEST_TAG_DATA" | jq -r '.object.sha')
+    TAGGED_COMMIT=$(echo "$TAG_DATA" | jq -r '.object.sha')
 
     # Extract voyager version from the release name
-    VOYAGER_VERSION=$(echo "$LATEST_RELEASE_NAME" | sed 's/v//')
+    VOYAGER_VERSION=$(echo "$RELEASE_NAME" | sed 's/v//')
```

Comment:
rename this to `VOYAGER_RELEASE_VERSION`

Reply:
Done